packages/base/src/formbuilder/objectform/vectorlayerform.ts

```diff
@@ -74,9 +74,38 @@ export class VectorLayerPropertiesForm extends LayerPropertiesForm {
     super.processSchema(data, schema, uiSchema);
 
     uiSchema['color'] = {
-      'ui:widget': 'color'
```

Comment:
This seems to break the color picker for the GeoJSON layer properties panel

Reply:
We may need some fancy new color widget here instead of the default one RJSF provides.

Let's probably keep that for a future PR